src/it/resources/com/google/checkstyle/test/chapter2filebasic/rule233nonascii/InputAvoidEscapedUnicodeCharacters.java

```diff
@@ -17,15 +17,15 @@ public Object fooString()
         String fakeUnicode = "asd\tsasd"; 
         String fakeUnicode2 = "\\u23\\u123i\\u";
         String content = "";
-        return "\ufeff" + content ; // byte order mark ok
+        /*byte order mark ok*/return "\ufeff" + content ;
```

Comment:
Yep. Thats why I make this change. If `feff` is marked as bad, a violation can be triggered like the line here https://github.com/checkstyle/checkstyle/blob/master/src/it/resources/com/google/checkstyle/test/chapter2filebasic/rule233nonascii/InputAvoidEscapedUnicodeCharacters.java#L15